exchanges/bybit/order_arguments.go

```diff
@@ -0,0 +1,135 @@
+package bybit
+
+import (
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+// DeriveSubmitOrderArguments returns a derived order arguments struct from an order.Submit for use in this package.
```

Comment:
🚧 
* `for use in this package` is both incorrect (since it's exported) and not really useful. If you mean it should be private, then it should be private.
* `derived` loses meaning in the middle of this sentence
```suggestion
// DeriveSubmitOrderArguments returns a PlaceOrderRequest from an order.Submit
```